assets/content/cookbook/Abstract types/working-around-iterator-limitations-as-generic-type-parameter.md

```diff
@@ -0,0 +1,37 @@
+[tags]: / "abstract-type,type-params,iterator"
+
+# Working Around Iterators as a Generic Type Parameter
```

Comment:
I tagged it as an abstract type article because I think Haxe's concept of it is less well known than iterators.